share/completions/firefox.fish

```diff
@@ -0,0 +1,42 @@
+# X11 options
+complete -c firefox -l display -r -d "X display to use"
+complete -c firefox -l sync -d "Make X calls synchronous"
+complete -c firefox -l g-fatal-warnings -d "Make all warnings fatal"
+
+# Firefox options
+complete -c firefox -s h -l help -d "Print this message"
+complete -c firefox -s v -l version -d "Print Firefox version"
+complete -c firefox -l full-version -d "Print Firefox version, build and platform build ids"
+complete -c firefox -s P -r -d "Start with <profile>"
+complete -c firefox -l profile -r -d "Start with profile at <path>"
+complete -c firefox -l migration -d "Start with migration wizard"
+complete -c firefox -l ProfileManager -d "Start with ProfileManager"
+complete -c firefox -l no-remote -d "Do not accept or send remote commands implies --new-instance"
+complete -c firefox -l new-instance -d "Open new instance, not a new window in running instance"
+complete -c firefox -l safe-mode -d "Disables extensions and themes for this session"
+complete -c firefox -l allow-downgrade -d "Allows downgrading a profile"
+complete -c firefox -l MOZ_LOG -r -d "Treated as MOZ_LOG=<modules> environment variable, overrides it"
+complete -c firefox -l MOZ_LOG_FILE -r -d "Treated as MOZ_LOG_FILE=<file> environment variable, overrides it"
+complete -c firefox -l headless -d "Run without a GUI"
+complete -c firefox -l jsdebugger -d "Open the Browser Toolbox. Defaults to the local build but can be overridden by a firefox path"
```

Comment:
There's a bug in firefox: --jsdebugger is listed twice. Maybe remove this one with the long description if the first sentence is enough for completions?